redash/models/__init__.py

```diff
@@ -187,11 +187,18 @@ def get_schema(self, refresh=False):
 
         if cache is None:
             query_runner = self.query_runner
-            schema = sorted(
-                query_runner.get_schema(get_stats=refresh), key=lambda t: t["name"]
-            )
+            schema = query_runner.get_schema(get_stats=refresh)
+
+            try:
+                out_schema = [{"name": i['name'], "columns": sorted(i['columns'])}
+                for i in sorted(schema, key=lambda x: x['name'])]
```

Comment:
Are you using `black`? The indentation here is confusing :thinking:

Reply:
I wasn't before. Just added it to my env.